sdk/python/endpoints/batch/deploy-pipelines/hello-batch/sdk-deploy-and-test.ipynb

```diff
@@ -543,7 +543,16 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "scoring_job = list(ml_client.jobs.list(parent_job_name=job.name))[0]\n",
+    "import time\n",
+    "\n",
+    "job_found = False\n",
+    "while not job_found:\n",
+    "    try:\n",
+    "        scoring_job = list(ml_client.jobs.list(parent_job_name=job.name))[0]\n",
+    "        job_found = True\n",
```

Comment:
Is this variable "job_found" used anywhere?

Reply:
We initialize "job_found" as False and then loop until we get the job successfully. We break the loop by setting "job_found" to True.